internal/pkg/api/server/templates/dependencies/dependencies.go

```diff
@@ -398,9 +407,23 @@ func apiHttpClient(envs env.Provider, logger log.Logger, debug, dumpHttp bool) c
 	if telemetry.IsDataDogEnabled(envs) {
 		transport = ddHttp.WrapRoundTripper(
 			transport,
+			ddHttp.WithBefore(func(request *http.Request, span ddtrace.Span) {
+				// We use "http.request" operation name for request to the API,
+				// so requests to other API must have different operation name.
+				span.SetOperationName("kac.api.client.http.request")
+				span.SetTag("http.host", request.URL.Host)
+				if dotPos := strings.IndexByte(request.URL.Host, '.'); dotPos > 0 {
+					// E.g. connection, encryption, scheduler ...
+					span.SetTag("http.hostPrefix", request.URL.Host[:dotPos])
+				}
+				span.SetTag(ext.EventSampleRate, 1.0)
```

Comment:
To je tu v PR 2x, znamena ze span bude `persisted`, tj. nezmaze sa po 15 minutach.
Potrebujeme to, ak chceme na span zavesit nejaky graf/monitor.

DD pouziva nejaku automatizaciu a niektore spans automaticky uklada:
- root span, long span, span with error ...
- my potrebujeme este niektore spans naviac